airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ReplicationJobOrchestrator.java

```diff
@@ -149,6 +153,15 @@ public Optional<String> runJob() throws Exception {
     return Optional.of(Jsons.serialize(replicationOutput));
   }
 
+  private AirbyteApiClient getAirbyteApiClient() {
+    // TODO make this actually functional
+    return new AirbyteApiClient(
+        new ApiClient().setScheme("http")
+            .setHost("airbyte-server-svc")
+            .setPort(8001)
+            .setBasePath("/api"));
+  }
```

Comment:
This is the main missing piece right now: getting it working and trying it out in a kubernetes environment with the container orchestrator.